tests/ui/test_home.py

```diff
@@ -3,15 +3,19 @@
 from pages.desktop.home import Home
 
 
+@pytest.mark.django_db
 @pytest.mark.nondestructive
-def test_there_are_ten_most_popular_extensions(base_url, selenium):
+def test_there_are_ten_most_popular_extensions(
+        base_url, selenium, live_server, initial_data):
```

Comment:
`live_server` should not be needed. We should override `base_url` and have that depend on `live_server`. That may require us modifying the scope of the `base_url` fixture.